### DISCUSSION TITLE
2022/07/06/horovod-multi-nodes/

### DISCUSSION THREAD

**Discussion by giscus** [xv44586/giscus#7] [Announcements]
# 2022/07/06/horovod-multi-nodes/

NLP | Machine Learning | Developer

https://xv44586.github.io/2022/07/06/horovod-multi-nodes/

**Comment by drinkingxi** [comment]
想请问一下，多机多卡的时候，没有InfiniBand网卡的话，只用普通以太网进行通信，是不是速度特别慢，还不如用单机多卡呀？还有别的办法可以提升多机之间的通信速度么？

**Reply by xv44586** [reply]
应该还是单机多卡更快，只是加速比没有InfiniBand 的大，好久没跟这方向了，现在一般都是DeepSpeed + Megatron 了，或者Jax
